### PR TITLE
feat(Monitor) : Add graphana dashboard for UPS Server

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/INTLY-1444

## What
- add graphana dashboard for UPS Server
 
## Verification Steps
Check the dashboard here: https://grafana-route-middleware-monitoring.apps.london-4bc4.openshiftworkshop.com/d/_cxgpLnZx/unifiedpush-server?refresh=10s&orgId=1

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

Screenshot 2019-07-11 at 06 26 45

NOTE: After the metrics be exported we will come back here to check if all are working as we expected. 
See https://issues.jboss.org/browse/INTLY-2668

### CODE REVIEW

File: deploy/monitor/push_grafana_dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,1419 @@
...
+              "seriesOverrides": [],
+              "spaceLength": 10,
+              "stack": false,
+              "steppedLine": false,
+              "targets": [
+                {
+                  "expr": "sum(rate(ups_msg_fails_total{namespace=\"unifiedpush\"}[1m]))*1000",
+                  "format": "time_series",
+                  "intervalFactor": 1,
+                  "legendFormat": "Faillures",
```
Comment (austincunningham) [inline deploy/monitor/push_grafana_dashboard.yaml]: typo

File: deploy/monitor/push_grafana_dashboard.yaml [hunk 2]
```diff
@@ -0,0 +1,1419 @@
...
+              "pointradius": 5,
+              "points": false,
+              "renderer": "flot",
+              "seriesOverrides": [],
+              "spaceLength": 10,
+              "stack": false,
+              "steppedLine": false,
+              "targets": [
+                {
+                  "expr": "sum(rate(ups_msg_total{namespace=\"unifiedpush\"}[1m]))*1000",
```
Comment (austincunningham) [inline deploy/monitor/push_grafana_dashboard.yaml]: not sure if \ escapes work, I have use ' for inner commas in the passed and they work fine. 
```suggestion
                  "expr": "sum(rate(ups_msg_total{namespace='unifiedpush'}[1m]))*1000",
```

File: deploy/monitor/push_grafana_dashboard.yaml [hunk 3]
```diff
@@ -0,0 +1,1419 @@
...
+              "pointradius": 5,
+              "points": false,
+              "renderer": "flot",
+              "seriesOverrides": [],
+              "spaceLength": 10,
+              "stack": false,
+              "steppedLine": false,
+              "targets": [
+                {
+                  "expr": "(api_requests_duration_seconds{job=\"example-unifiedpushserver-unifiedpush\"})",
```
Comment (austincunningham) [inline deploy/monitor/push_grafana_dashboard.yaml]: ^^

**Review by austincunningham — COMMENTED** [review]
A lot of your charts don't appear to have any data points , may be an issue with your expressions for these

File: deploy/monitor/push_grafana_dashboard.yaml (hunk 1 above)
Comment (camilamacedo86) [reply deploy/monitor/push_grafana_dashboard.yaml]: 👍

File: deploy/monitor/push_grafana_dashboard.yaml (hunk 2 above)
Comment (camilamacedo86) [reply deploy/monitor/push_grafana_dashboard.yaml]: 👍

File: deploy/monitor/push_grafana_dashboard.yaml (hunk 3 above)
Comment (camilamacedo86) [reply deploy/monitor/push_grafana_dashboard.yaml]: 👍

**Comment by camilamacedo86** [comment]
HI @austincunningham,

The charts which have no data now and it is because of the metrics are not export yet in the UPS Server. These values are specified and defined here as https://issues.jboss.org/browse/INTLY-2642. Our approach is to leave it done and came back and do any adjustment it may be required in this task https://issues.jboss.org/browse/INTLY-2668.

**Review by wei-lee — APPROVED** [review]

**Pull request merged**